IdentityServer/v7/docs/content/quickstarts/3_api_access.md

```diff
@@ -91,7 +88,6 @@ builder.Services.AddAuthentication(options =>
         options.Scope.Add("openid");
         options.Scope.Add("profile");
         options.Scope.Add("api1");
-        options.Scope.Add("offline_access");
         options.Scope.Add("verification");
```

Comment:
what is this scope - do we need it?

Reply:
It is a scope added in the "Further experiments" section of the "Interactive Applications with ASP.NET Core" qs.